lib/Secrets/interactionparameters.h

```diff
@@ -59,28 +59,37 @@ class SAILFISH_SECRETS_API InteractionParameters {
     Q_ENUM(EchoMode)
 
     enum Operation {
-        UnknownOperation = 0,
-
-        RequestUserData  = 1,   // usually used in conjunction with StoreSecret, i.e. store data requested from user.
-
-        CreateCollection = 2,
-        UnlockCollection = 4,
-        DeleteCollection = 8,
-
-        ReadSecret       = 16,
-        StoreSecret      = 32,
-        DeleteSecret     = 64,
-
-        Encrypt          = 128,
-        Decrypt          = 256,
-        Sign             = 512,
-        Verify           = 1024,
-        DeriveDigest     = 2048,
-        DeriveMac        = 4096,
-        DeriveKey        = 8192,
+        UnknownOperation    = 0,
+
+        RequestUserData     = 1<<0,   // usually used in conjunction with StoreSecret, i.e. store data requested from user.
```

Comment:
In other places, you use spaces next to the `<<` operator. For consistency, I suggest `1 << 0` here too.

Reply:
Done